internal/statestore/instrumented.go

```diff
@@ -141,3 +141,24 @@ func (is *instrumentedService) NewMutex(key string) RedisLocker {
 	defer span.End()
 	return is.s.NewMutex(key)
 }
+
+// IndexBackfill adds the backfill to the index.
+func (is *instrumentedService) IndexBackfill(ctx context.Context, backfill *pb.Backfill) error {
+	_, span := trace.StartSpan(context.Background(), "statestore/instrumented.IndexBackfill")
```

Comment:
why is context.Background is used instead of ctx?

Reply:
probably slipped in conflict resolution. Will fix it